packages/react-components/src/organisms/PerformanceCard.tsx

```diff
@@ -0,0 +1,42 @@
+import {
+  PerformanceMetricByDocumentType,
+  UserProductivityPerformance,
+} from '@asap-hub/model';
+import { CaptionItem } from '../molecules';
+import CaptionCard from './CaptionCard';
+
+type PerformanceCardProps = {
+  performance: PerformanceMetricByDocumentType | UserProductivityPerformance;
+  type: 'by-document' | 'by-output';
+};
+
+const PerformanceCard: React.FC<PerformanceCardProps> = ({
+  performance,
+  type,
+}) => {
+  if (type === 'by-document') {
```

Comment:
I understand this is needed because the Team Productivity and Team Collaboration tables use the same performance card. The rest are all different though so I was torn between whether we need it or not. I guess the complexity is not increased significantly and we don't have duplicate code in 2 places so that's ok.